src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
Recording does not start before. Event for keyup is added same here as it was added before so it's the same. I check if we should start recording in the event keydown, you are checking it elsewhere. I don't agree with it that it's hacky, your approach is different and more complex. One other thing that I don't understand is how is this related to the popover escape, which is the name of this PR?